.github/workflows/release-azure-pipelines.yml

```diff
@@ -93,40 +93,268 @@ extends:
+            inputs:
+              rootFolderOrFile: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-win10-x64-signed/AzureAuth
+              includeRootFolder: false
+              archiveType: zip
+              archiveFile: azureauth-${{ parameters.version }}-win10-x64.zip
+          - task: Bash@3
+            displayName: Prepare osx-x64 executables
+            inputs:
+              targetType: inline
+              workingDirectory: $(Build.ArtifactStagingDirectory)
+              script: |
+                cd azureauth-${{ parameters.version }}-osx-x64-signed/AzureAuth
+                chmod +x azureauth createdump *.dylib
+          - task: ArchiveFiles@2
+            displayName: Create osx-x64 archive
+            inputs:
+              rootFolderOrFile: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-osx-x64-signed/AzureAuth
+              includeRootFolder: false
+              archiveType: tar
+              tarCompression: gz
+              archiveFile: azureauth-${{ parameters.version }}-osx-x64.tar.gz
+          - task: Bash@3
+            displayName: Prepare osx-arm64 executables
+            inputs:
+              workingDirectory: $(Build.ArtifactStagingDirectory)
+              targetType: inline
+              script: |
+                cd azureauth-${{ parameters.version }}-osx-arm64-signed/AzureAuth
+                chmod +x azureauth createdump *.dylib
+          - task: ArchiveFiles@2
+            displayName: Create osx-arm64 archive
+            inputs:
+              rootFolderOrFile: $(Build.ArtifactStagingDirectory)/azureauth-${{ parameters.version }}-osx-arm64-signed/AzureAuth
+              includeRootFolder: false
+              archiveType: tar
+              tarCompression: gz
+              archiveFile: azureauth-${{ parameters.version }}-osx-arm64.tar.gz
 
-            - task: DotNetCoreCLI@2
-              displayName: Test
-              inputs:
-                command: test
-                arguments: --configuration release --no-restore
-            
-            - task: DotNetCoreCLI@2
-              displayName: Build artifacts
-              env:
-                ADO_TOKEN: $(System.AccessToken)
-              inputs:
-                command: publish
-                projects: src/AzureAuth/AzureAuth.csproj
-                arguments: -p:Version=${{ parameters.version }} --configuration release --self-contained true --runtime ${{ config.runtime }} --output dist/${{ config.runtime }}
-                publishWebProjects: false
-                zipAfterPublish: false
-                modifyOutputPath: true
\ No newline at end of file
+    - stage: release
+      displayName: Release
+      dependsOn: package
+      jobs:
+      - job: approval
+        displayName: Manual Approval
+        pool: server
+        timeoutInMinutes: 5760 # job times out in 4 days
+        steps:
+        - task: ManualValidation@0

```

Comment:
Ooo this is a clever addition. I suspect that in the future, when release pipeline templates become a thing, that this could end up being a lockbox approval instead of a pipeline approval (if we're required to use MOBR). But we'll see what that looks like when we get there!